Assignment2/belenfh999-fromJSONtoJSON-LD.jsonld

```diff
@@ -0,0 +1,39 @@
+{
+	"@context":{
+		"vocab":"http://example.com/semweb/",
+		"fullName":"vocab:fullName",
+		"hometown":"vocab:hometown",
+		"age":"vocab:age",
+		"almaMater":"vocab:almaMater",
+		"parentsName":"vocab:parentsName",
+		"subjects":"vocab:subjects",
+		"name":"subjects:name",
+		"teachers":"subjects:teachers",
+		"description":"subjects:description"

```

Comment:
"subjects" prefix is not defined. Define it or reuse already defined "vocab" instead.